homepage-news.md

```diff
@@ -0,0 +1,149 @@
+# Homepage News Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/pages**   
+
+Provide access to the list of all static pages (currently home page news only). These are pages not specific to any items. The design of the REST contract should allow it to be extended to e.g. images as well.
```

Comment:
I know Lieven said it wasn't with high priority. But reading this text, it seems to be in an earlier stage.

Reply:
The contract is actually not in an early phase.
The contract was designed to support the various pages without changes to the contract, but the Java implementation will be limited to the home page news for DSpace 7. This may be extended in DSpace 8, and the goal is to achieve this without changes to the contract

Reply:
ok, thank you for the clarification!

Reply:
This contract has been designed specifically to ensure institutions can easily add a page with minimal effort.
This is exactly the reason for the approach where static pages can be created/downloaded.
So contrary to the old JSPUI config you pointed at, the ability to create static pages won't require any changes in the backend.
All that's required is a `POST` of a new page with a different name.